test/models/repository_test.rb

```diff
@@ -78,28 +78,21 @@ def test_docker_repository_docker_upstream_name_format
       @repo.unprotected = true
       @repo.content_type = 'docker'
       @repo.download_policy = nil
-      @repo.docker_upstream_name = 'valid'
-      assert @repo.valid?
-      @repo.docker_upstream_name = 'Invalid'
-      refute @repo.valid?
-      @repo.docker_upstream_name = '-_ok/valid'
-      assert @repo.valid?
-      @repo.docker_upstream_name = 'Invalid/valid'
-      refute @repo.valid?
-      @repo.docker_upstream_name = 'Invalid/Invalid'
-      refute @repo.valid?
-      @repo.docker_upstream_name = 'abcd/.-_'
-      assert @repo.valid?
-      @repo.docker_upstream_name = 'abc/valid'
-      refute @repo.valid?
-      @repo.docker_upstream_name = 'abcd/ab'
-      refute @repo.valid?
-      @repo.docker_upstream_name = '/valid'
-      refute @repo.valid?
-      @repo.docker_upstream_name = 'thisisnotvalidbecauseitistoolong/valid'
-      refute @repo.valid?
-      @repo.docker_upstream_name = 'valid/thisisnotvalidbecauseitistoolong'
-      refute @repo.valid?
+      %w( valid abc/valid
+          thisisareallylongbutstillvalidname soisthis/thisisareallylongbutstillvalidname
+          single/slash ).each do |name|
```

Comment:
This test would be a bit easier to read if this list and the one below were named arrays before being used. e.g

```
valid = %w()
invalid = %w()
```